kimchi/src/verifier.rs

```diff
@@ -433,7 +433,25 @@ where
             .chain((0..COLUMNS).map(Column::Witness))
             .chain((0..COLUMNS).map(Column::Coefficient))
             .chain((0..PERMUTS - 1).map(Column::Permutation))
-            {
+            .chain(
+                index
+                    .lookup_index
+                    .as_ref()
+                    .map(|li| {
+                        (0..li.lookup_info.max_per_row + 1)
+                            .map(Column::LookupSorted)
+                            .chain([Column::LookupAggreg, Column::LookupTable].into_iter())
+                            .chain(
+                                li.runtime_tables_selector
+                                    .as_ref()
+                                    .map(|_| [Column::LookupRuntimeTable].into_iter())
+                                    .into_iter()
+                                    .flatten(),
+                            )
+                    })
+                    .into_iter()
+                    .flatten(),
+            ) {
```

Comment:
I'm puzzled as to how the lookup tests worked if we were not checking the lookup parts in the verification of the aggregated evaluation proof?

Reply:
We weren't using this `combined_inner_product` *except* on the pickles side. In practice, this means that everything appeared to work on the kimchi side, but pickles would be broken.

Reply:
that's what I had guessed :D good job for spotting this : )